src/types/onyx/Transaction.ts

```diff
@@ -232,6 +300,8 @@ type Transaction = OnyxCommon.OnyxValueWithOfflineFeedback<
         /** Indicates transaction loading */
         isLoading?: boolean;
 
+        originalSpotnanaPayload?: SpotnanaPayload;
```

Comment:
NAB but let's add a description here?

Reply:
Turns out it was dead code. Was from initial implementation. 
Removed it